eclipse-theia/theia
# core: remove deprecated `shell` emitters

#### What it does


The pull-request removes the deprecated `currentChanged` and `activeChanged` signal emitters from `application-shell` in favor of `onDidChangeCurrentWidget` and `onDidChangeActiveWidget` respectfully. The emitters are removed since they were deprecated for some time, and any reference to them are updated.

#### How to test


There should not be any change of functionality, the repo should build and pass tests, and the events should properly fire.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
